docker/generate_build_matrix.py

```diff
@@ -18,17 +18,17 @@
 import tabulate
 import yaml
 
-LATEST_PYTHON_VERSION = '3.10'
-PRODUCTION_PYTORCH_VERSION = '2.1.2'
+LATEST_PYTHON_VERSION = '3.11'
+PRODUCTION_PYTORCH_VERSION = '2.2.0'
```

Comment:
Let's leave latest as 2.1.2 until we let 2.2.0 burn-in for a bit

Reply:
fixed

Reply:
fixed